package.json

```diff
@@ -17,10 +18,12 @@
     "use-react-router-breadcrumbs": "^4.0.1"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "npm run build:glean && react-scripts start",
```

Comment:
I'm not sure we should add this command to each of the `scripts`. I think we would be better off leaving it in just the `build` command and adding a new step in the README for set up. The new step should be after `npm install` and just be something like `npm run build:glean` before they try and run locally or build or deploy or etc.

Reply:
Got it. Made this change in the latest commit.